### ISSUE TITLE
Possible rare bug with non-Vortex managed mods.

### ISSUE THREAD

**Issue by odelphi** [Nexus-Mods/Vortex#6765]
I think this is a weird, quirky rare bug, but it may be expected behavior. I accidentally reinstalled a mod in Witcher 3 that ,unbeknownst to me, was not originally installed by Vortex. Prolly installed with WMM.   It installed fine and the script merger worked fine (unfortunately I didn't think to run the game), but something weird and quirky happened.  All my mods that shared a script with this new mod started appearing twice in my load order - once as a regular Vortex mod and again as a "not managed by Vortex" mod.  It ended up being about 12-13 mods.  I quickly uninstalled the non-Vortex mod, but nothing changed.  I had to reinstall ALL my mods that were affected by this issue and rerun the script merger.  It is fixed now and I installed a Vortex version of the new mod and everything is working fine.

1. install several vortex managed mods. 
2.  Run the script merger and merge scripts as appropriate
3.  Install a mod not managed by Vortex that shares scripts with Vortex-managed mods.

**Platform (please complete the following information):**
 - OS: Windows 10
 - Vortex Version: 1.2.17

**Additional context**
The non-managed mod was "Autoloot All-in-one".

**Comment by IDCs** [comment]
Hey @odelphi, so I've been trying to reproduce this with no luck; are you able to do so consistently? if so could you possibly download the screen to gif application (https://www.screentogif.com/) and record your screen while you reproduce it ?

Your vortex.log files would be greatly appreciated as well - you can find those in "%appdata%/vortex"

**Comment by IDCs** [comment]
BTW, I've tested the autoloot mod and it will not install the key bindings correctly when using Vortex as it's not using the expected packaging pattern; I'll let our CM team know so they contact the mod author so he can hopefully release a new Vortex friendly version.

**Comment by odelphi** [comment]
Just FYI:  I am having an issue (I reported it) with my keybindings screen (in game) being blank.  I uninstalled ALL my mods and the keybindings came back.  Based on your comment, I installed JUST the AutoLoot mod and the keybindings are still working fine.  So I am thinking it is not the Autoloot mod causing the missing keybindings, but I am going to test it some more to see when the keybindings disappear.  Thanks for your help
